src/num_methods/binary_search.md

```diff
@@ -40,7 +40,7 @@ Logarithmic number of steps is drastically better than that of linear search. Fo
 
 ### Lower bound and upper bound
 
-It is often convenient to find the position of the first element that is not less than $k$ (called the lower bound of $k$ in the array) or the position of the first element that is greater than $k$ (called the upper bound of $k$) rather than the exact position of the element.
+It is often convenient to find the position of the first element that is less than $k$ (called the lower bound of $k$ in the array) or the position of the first element that is greater than $k$ (called the upper bound of $k$) rather than the exact position of the element.
```

Comment:
```suggestion
It is often convenient to find the position of the first element that is greater or equal than $k$ (called the lower bound of $k$ in the array) or the position of the first element that is greater than $k$ (called the upper bound of $k$) rather than the exact position of the element.
```
You probably meant to do this change?

Reply:
@adamant-pwn yah.